.github/workflows/test-python.yml

```
@@ -190,8 +190,30 @@ jobs:
       - name: Validate version
         run: |
           $pythonVersion = (python --version)
-          if ("$pythonVersion" -NotMatch "${{ matrix.python }}"){
-            Write-Host "The current version is $pythonVersion; expected version is ${{ matrix.python }}"
+          if ("$pythonVersion" -NotMatch "${{ matrix.python-version }}"){
+            Write-Host "The current version is $pythonVersion; expected version is ${{ matrix.python-version }}"
+            exit 1
+          }
+          $pythonVersion
+        shell: pwsh
+  setup-python-multiple-versions:
```

nitpick: add space between jobs
```suggestion
        shell: pwsh
        
  setup-python-multiple-versions:
```